include/qpid/dispatch/tls.h

```diff
@@ -0,0 +1,183 @@
+#ifndef __tls_h__
```

Comment:
Can we please rename `tls.h` to `tls_common.h` ?

Reply:
Done.